package/src/components/ImageGallery/components/ImageGrid.tsx

```diff
@@ -77,14 +79,16 @@ const GridImage = <Us extends DefaultUserType = DefaultUserType>({
 
   return (
     <TouchableOpacity onPress={selectAndClose}>
-      <ImageBackground
+      <CachedImageBackground
```

Comment:
@vishalnarkhede do you know what this component is doing? I cached this background image as well but tbh I couldnt use it to reproduce anything.